config/default/Node.json

```diff
@@ -3,5 +3,6 @@
 	"sync_loss_delay": 180,
 	"max_fork_length": 1000,
 	"validate_interval_ms": 500,
+	"verify_vdf_cpuopencl": false,
```

Comment:
Not really needed here since the default in Node.vni already is `false`.

Reply:
Done.